src/common/pubsub.ts

```diff
@@ -333,10 +333,10 @@ export const notifyFreeformContentRequested = async (
   freeform: ChangeMessage<FreeformPost>,
 ): Promise<void> =>
   notifyContentRequested(logger, {
-    id: freeform.payload.after.id,
-    content: freeform.payload.after.content,
-    title: freeform.payload.after.title,
-    post_type: freeform.payload.after.type,
+    id: freeform.payload.after!.id,
+    content: freeform.payload.after!.content,
+    title: freeform.payload.after!.title,
+    post_type: freeform.payload.after!.type,
   });
```

Comment:
```ts
  const after = freeform.payload.after!
  notifyContentRequested(logger, {
    id: after.id,
    content: after.content,
    title: after.title,
    post_type: after.type,
  });
```

Could this work? To not having to do `!` each time?

Reply:
yeah, sure 38d60bb170ea246859c834afbe4ecf7a24ed6eac